src/Map/src/Live/ComponentWithMapTrait.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Live;
+
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\Map\Map;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+use Symfony\UX\TwigComponent\Attribute\PostMount;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+trait ComponentWithMapTrait
+{
+    /**
+     * @internal
+     */
+    #[LiveProp(hydrateWith: 'hydrateMap', dehydrateWith: 'dehydrateMap')]
```

Comment:
```suggestion
    #[LiveProp(hydrateWith: 'hydrateMap', dehydrateWith: 'dehydrateMap')]
```
Should be  private i think, and we maybe should create a dedicated hydration extension here.. wdyt ?

Reply:
I put it public otherwise the Live hydration process was not working due to visibility issue. Maybe I can try to add a setter?

Reply:
Let's discuss all this tomorrow :)